docs/tutorials/13_quantum_random_access_optimizer.ipynb

```diff
@@ -0,0 +1,892 @@
+      "SolutionSample(x=array([1, 1, 1, 0, 0, 1]), fval=6.0, probability=0.0212, status=<OptimizationResultStatus.SUCCESS: 0>)\n",
+      "SolutionSample(x=array([0, 1, 1, 1, 1, 0]), fval=6.0, probability=0.020799999999999996, status=<OptimizationResultStatus.SUCCESS: 0>)\n",
+      "SolutionSample(x=array([1, 0, 0, 0, 0, 1]), fval=6.0, probability=0.022, status=<OptimizationResultStatus.SUCCESS: 0>)\n",
+      "SolutionSample(x=array([1, 0, 1, 0, 0, 0]), fval=6.0, probability=0.0211, status=<OptimizationResultStatus.SUCCESS: 0>)\n",
+      "SolutionSample(x=array([0, 1, 0, 1, 1, 1]), fval=6.0, probability=0.0223, status=<OptimizationResultStatus.SUCCESS: 0>)\n",
+      "SolutionSample(x=array([1, 0, 1, 0, 1, 1]), fval=6.0, probability=0.0213, status=<OptimizationResultStatus.SUCCESS: 0>)\n",
+      "SolutionSample(x=array([0, 1, 0, 1, 0, 0]), fval=6.0, probability=0.0214, status=<OptimizationResultStatus.SUCCESS: 0>)\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(f\"The number of distinct samples is {len(results.samples)}.\")\n",
+    "print(\"Top 10 samples with the largest fval:\")\n",
+    "for sample in results.samples[:10]:\n",
+    "    print(sample)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Alternative: Solve the Problem in Two Explicit Steps\n",
+    "\n",
+    "In the previous part of this tutorial, we utilized the `qrao.solve()` method, which solved the encoded problem (the ground state of the relaxed Hamiltonian) and performed rounding to map the ground state results back to a solution of the original problem. However, it is also possible to explicitly break down the calculation into these two distinct steps. This can be beneficial, especially when comparing solutions obtained across multiple rounding schemes applied to a candidate ground state.\n",
+    "\n",
+    "In this section, we will explore how to perform each of these steps explicitly.\n",
+    "\n",
+    "## Manually solve the relaxed problem.\n",
+    "\n",
+    "Let's start by invoking the `qrao.solve_relaxed()` method to directly solve the relaxed problem encoded by `QuantumRandomAccessEncoding`.\n",
+    "This method allows us to focus solely on solving the relaxed problem without performing rounding.\n",
+    "\n",
+    "By invoking `qrao.solve_relaxed()`, we obtain two essential outputs:\n",
+    "\n",
+    "- `MinimumEigensolverResult`: This object contains the results of running the minimum eigen optimizer such as the VQE on the relaxed problem. It provides information about the ground state, eigenvalues, and other relevant details. You can refer to the Qiskit [documentation](https://qiskit.org/documentation/stubs/qiskit.algorithms.eigensolvers.EigensolverResult.html#qiskit.algorithms.eigensolvers.EigensolverResult) for a comprehensive explanation of the entries within this object.\n",
```

Comment:
I think the link here should be for the MinimumEigensolverResult right, not the EigenSolverResult. It should now say "Qiskit Algorithms" too not Qiskit. Since it says `groundstate, eigenvalues` where the latter is plural I figured rather than make a suggestion just comment. The result also no longer contains the `groundstate` - was true of the QI based algos but that changed with primitives.